.github/workflows/test.yml

```diff
@@ -53,6 +56,7 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+          allow-prereleases: true
```

Comment:
I would also not `allow-prereleases` it is a whole other discussion than supporting 3.12.

Reply:
It won't make any difference to this PR: with GHA it uses the latest tagged releases (unlike Travis CI, which used to do nightly builds), so 3.12.0 now, 3.12.1 soon.

But I can remove it, it was needed when this PR was opened back in September before 3.12.0 final was released, to allow installing latest alpha/beta/RC.

```suggestion
```